modules/tinymce/src/core/test/ts/browser/DragnDropCEFTest.ts

```diff
@@ -0,0 +1,62 @@
+import { Mouse, UiFinder, Waiter } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { SugarLocation } from '@ephox/sugar';
+import { TinyAssertions, TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.DragnDropCEFTest', () => {
+  const getBaseCEFElement = (name: string) =>
+    `<div class="${name}" style="margin: 40px; width: 1110px; height: 120px;" contenteditable="false">${name}</div>`;
+
+  const getContentWithCefElements = (elementsNames: string[]): string => {
+    if (!Arr.contains(elementsNames, 'toDrag') || !Arr.contains(elementsNames, 'destination')) {
+      // eslint-disable-next-line no-throw-literal
+      throw new Error('This function require to have an element named toDrag and one destination');
+    }
+    return `<div>${Arr.foldl(elementsNames, (acc, elementName) =>
+      acc + getBaseCEFElement(elementName)
+    , '')}</div>`;
+  };
+
+  const hook = TinyHooks.bddSetup<Editor>({
```

Comment:
If keeping this, why do you need a full editor and not a lightweight one? This doesn't appear to use the UI at all so this should be using `bddSetupLight` instead.

Reply:
changed